src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js

```diff
@@ -49,7 +49,7 @@ export const ProgramsPageWrapper = styled.div`
         padding: 5px;
         color: ${props => props.theme.text};
         margin: auto;
-        background: ${props => props.theme.DarkTheme ? "rgba(10, 10, 10, 1)" : "rgba(247, 247, 247, 1)"};
+        background: ${props => props.theme.DarkTheme ? "rgba(152, 152, 152, 1)" : "rgba(247, 247, 247, 1)"};
```

Comment:
This is not the solution we intend to have.
Use different versions of MLH and the Layer5 logo for the light/dark theme.
Layer5 logo could be found under the assets directory, and the MLH logo could be downloaded from the issue, renamed, and added to the repo.

Reply:
Got it, @leecalcote also mentioned background was too light, have, changed, but will change the logos as well.  Thanks @Nikhil-Ladha

Reply:
I think he meant for the light mode. Not for the dark mode.